common/src/main/java/com/microsoft/identity/common/internal/commands/parameters/RefreshInTokenCommandParameters.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.identity.common.internal.commands.parameters;
+
+import com.google.gson.annotations.Expose;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder(toBuilder = true)
+public class RefreshInTokenCommandParameters extends SilentTokenCommandParameters {
```

Comment:
I was really hoping that we'd get away from implementing an entirely different class for this kind of thing.  This is a type that's essentially being used as a flag, and when done in this way it's really hard for a developer to know or discover what is going on.  Why not have a flag instead?